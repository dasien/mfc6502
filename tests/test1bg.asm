; Perform simple addition

LDA #01
STA $01
LDA #02
STA $02
ADC $01
STA $03
