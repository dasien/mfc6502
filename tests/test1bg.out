0100 A9 01
0102 85 01
0104 A9 02
0106 85 02
0108 65 01
010A 85 03
010C 0D 
010D 0A 
010E 00 
010F 57 
0110 57 
0111 57 
0112 57 
0113 57 
0114 57 
0115 57 
0116 57 
0117 57 
0118 57 
0119 57 
011A 57 
011B 57 
011C 57 
011D 57 
011E 57 
011F 42 
0120 42 
0121 42 
0122 42 
0123 42 
0124 42 
0125 42 
0126 42 
0127 42 
0128 42 
0129 42 
012A 42 
012B 42 
012C 42 
012D 42 
012E 42 
012F 57 
0130 57 
0131 57 
0132 57 
0133 57 
0134 57 
0135 57 
0136 57 
0137 57 
0138 57 
0139 57 
013A 57 
013B 57 
013C 57 
013D 57 
013E 57 
013F 4B 
0140 51 
0141 43 
0142 43 
0143 42 
0144 42 
0145 52 
0146 52 
0147 50 
0148 50 
0149 50 
014A 50 
014B 50 
014C 50 
014D 50 
014E 50 
014F 4B 
0150 51 
0151 43 
0152 43 
0153 42 
0154 42 
0155 52 
0156 52 
0157 50 
0158 50 
0159 50 
015A 50 
015B 50 
015C 50 
015D 50 
015E 50 
015F 00 
